src/+otp/+robertson/RobertsonParameters.m

```diff
@@ -0,0 +1,7 @@
+classdef RobertsonParameters
+    properties
+        K1

```

Comment:
These are all reaction coefficients, so they are also 

```%MATLAB ONLY: (1,1) {..........}```` finite, and positive